sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4082,7 +4138,7 @@ public partial class LinuxPatchSettings
     {
         public LinuxPatchSettings() { }
         public Azure.ResourceManager.Compute.Models.LinuxPatchAssessmentMode? AssessmentMode { get { throw null; } set { } }
-        public Azure.ResourceManager.Compute.Models.LinuxVmGuestPatchAutomaticByPlatformRebootSetting? AutomaticByPlatformRebootSetting { get { throw null; } set { } }
```

Comment:
System.Nullable<Azure.ResourceManager.Compute.Models.LinuxVmGuestPatchAutomaticByPlatformRebootSetting> Azure.ResourceManager.Compute.Models.LinuxPatchSettings.AutomaticByPlatformRebootSetting.get()' does not exist in the implementation but it does exist in the contract.	Azure.ResourceManager.Compute	

How do i resolve above build error?

Reply:
See this comment: https://github.com/Azure/azure-sdk-for-net/pull/35676/files#r1173384888

Reply:
resolved